content/wasm-functions/using-cron-jobs.md

```diff
@@ -0,0 +1,237 @@
+title = "Scheduling Tasks with Cron Jobs in Spin"
+template = "functions_main"
+date = "2025-03-25T00:00:01Z"
+enable_shortcodes = true
+
+---
+- [Prerequisites](#prerequisites)
+- [Introduction to Cron Jobs](#introduction-to-cron-jobs)
+  - [What Are Cron Jobs?](#what-are-cron-jobs)
+- [Create a Spin Application](#create-a-spin-application)
+  - [Deploy Spin Application to Fermyon Wasm Functions](#deploy-spin-application-to-fermyon-wasm-functions)
+- [Scheduling a Cron Job for Your Spin Application](#scheduling-a-cron-job-for-your-spin-application)
+- [Managing Cron Jobs](#managing-cron-jobs)
+- [Delete Cron Job](#delete-cron-job)
+- [Next Steps](#next-steps)
+  
+This tutorial guides you through the process of scheduling HTTP requests in a Spin application running on Fermyon Wasm Functions using cron jobs. The `spin aka crons` command allows applications to receive scheduled HTTP triggers without relying on external services. In this tutorial, you'll learn how to define and manage cron jobs using the `spin aka crons` command.
+
+# Prerequisites
+Before proceeding, ensure you have the following:
+
+- Spin and the `spin aka` plugin installed on your machine ([Installation Guide](./quickstart.md))
+- Access to Fermyon Wasm Functions preview ([Sign Up Form](https://fibsu0jcu2g.typeform.com/fwf-preview))
+
+# Introduction to Cron Jobs
+
+## What Are Cron Jobs?
+
+Cron jobs are scheduled tasks that run at specified intervals. They are commonly used for automating repetitive tasks such as data processing, report generation, and background maintenance. Other potential use cases include: 
+
+* Automated data backups
+* Periodic API polling
+* Log file rotation
+* Scheduled notifications
+
+# Create a Spin Application 
+
+You must have a Spin application deployed to Fermyon Wasm Functions to associate a cron job. We'll create a new Spin application using the `http-js` template, which we will then invoke on a schedule using a cron job in later steps:
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin new -t http-js --accept-defaults hello-cron-job
+
+$ cd hello-cron-job
+
+$ npm install
+```
+
+We'll go ahead add a path that will be used our cron job in later steps:
+
+```toml
+[[trigger.http]]
+route = "/cron-me"
+component = "hello-cron-job"
+```
+
+> Note that you can have multiple cron jobs per Spin application as long as the combination of schedule and path_and_query is unique. We will review how to set those values later in the tutorial.  
```

Comment:
This seems a bit disconnected from "add a trigger."  It feels like you're setting up for a mental model of "multiple cron jobs per app, each cron job handled by a different component, those components on different triggers, cron jobs mapped to components via their routes" but it's not explicit, and the "each trigger points at the same component as the everything route and we route within the component" doesn't draw out that structure.

Perhaps move this down to where you talk about creating cron jobs, and if you want to talk about how to structure a multi-job app then do so there?  And focus on the simple "get something working" path here?  Or do you feel that leads readers to a dead end?

Reply:
Agreed - I've moved it down to the section where I talk about the required parameters for the `spin aka crons create` subcommand. I decided to not expand into multi-job to keep the tutorial manageable, but I like the idea of creating a more complex example for Spin Hub or fwf-examples we can link to in this tutorial for those wanting a more realistic example.